client/src/routes.tsx

```diff
@@ -4,7 +4,7 @@ export const slugs = {
   grant: `/grants/:id`,
   edit: `/grants/:id/edit`,
   newGrant: `/grants/new`,
-  round: `/rounds/:id`,
+  round: `/:chainId/rounds/:roundId`,
```

Comment:
wdyt if we set it as `/chains/chainId/rounds/roundId` just to follow the same king of REST pattern?

Reply:
Agreed, can we update and merge this bad boy 💥 WDYT @schultztimothy